snprc_ehr/src/org/labkey/snprc_ehr/controllers/FeeScheduleController.java

```diff
@@ -101,14 +103,27 @@ public ModelAndView getView(FeeScheduleImportForm form, boolean reshow, BindExce
         {
             String fileSize;
             String dateString;
+            File fileToVerify = null;
             File file = null;
             SimpleDateFormat formatString = new SimpleDateFormat("MM/dd/yyyy hh:mm");
 
             List<String> tabPageNames;
             FeeScheduleExcelParser fsep;
             if (form.getFilePath() != null)
             {
-                file = new File(form.getFilePath());
+                //file path here is constructed from the pipeline, and contains pipeline root + actual file.
+                fileToVerify = new File(form.getFilePath());
```

Comment:
@spamhurts Does the .xlsx/.xls file here expected to reside directly under the pipeline root? or they are under a directory (or nested directories) within a pipeline root?

Reply:
They are currently getting copied into the pipeline root

Reply:
Got it! Thank you!